docs/outlook/mobile-log-appointments.md

```diff
@@ -0,0 +1,544 @@
+                  </Icon>
+                  <Action xsi:type="ExecuteFunction">
+                    <FunctionName>logCRMEvent</FunctionName>
+                  </Action>
+                </Control>
+              </ExtensionPoint>
+            </MobileFormFactor>
+          </Host>
+        </Hosts>
+        <Resources>
+          <bt:Images>
+            <bt:Image id="icon-16" DefaultValue="https://contoso.com/assets/icon-16.png"/>
+            <bt:Image id="icon-32" DefaultValue="https://contoso.com/assets/icon-32.png"/>
+            <bt:Image id="icon-48" DefaultValue="https://contoso.com/assets/icon-48.png"/>
+            <bt:Image id="icon-80" DefaultValue="https://contoso.com/assets/icon-80.png"/>
+          </bt:Images>
+          <bt:Urls>
+            <bt:Url id="residFunctionFile" DefaultValue="https://contoso.com/commands.html"/>
+          </bt:Urls>
+          <bt:ShortStrings>
+            <bt:String id="residDescription" DefaultValue="Log appointment notes and other details to Contoso CRM."/>
+            <bt:String id="residLabel" DefaultValue="Log to Contoso CRM"/>
+          </bt:ShortStrings>
+          <bt:LongStrings>
+            <bt:String id="residTooltip" DefaultValue="Log notes to Contoso CRM for this appointment."/>
+          </bt:LongStrings>
+        </Resources>
+      </VersionOverrides>
+    </VersionOverrides>
+    ```
+
+> [!TIP]
+> To learn more about manifests for Outlook add-ins, see [Outlook add-in manifests](manifests.md) and [Add support for add-in commands for Outlook Mobile](add-mobile-support.md).
+
+### Capture appointment notes
+
+In this section, learn how your add-in can extract appointment details when the user selects the **Log** button.
+
+1. From the same quick start project, open the file **./src/commands/commands.js** in your code editor.
+
+1. Replace the entire content of the **commands.js** file with the following JavaScript.
+
+    ```js
+    var event;
+
+    Office.initialize = function (reason) {
+      // Add any initialization code here.
+    };
+
+    function logCRMEvent(appointmentEvent) {
+      event = appointmentEvent;
+      console.log(`Subject: ${Office.context.mailbox.item.subject}`);
+      Office.context.mailbox.item.body.getAsync(
+        "html",
+        { asyncContext: "This is passed to the callback" },
+        function callback(result) {
+          if (result.status === Office.AsyncResultStatus.Succeeded) {
+            event.completed({ allowEvent: true });
+          } else {
+            console.error("Failed to get body.");
+            event.completed({ allowEvent: false });
+          }
+        }
+      );
+    }
+
+    function getGlobal() {
+      return typeof self !== "undefined"
+        ? self
+        : typeof window !== "undefined"
+        ? window
+        : typeof global !== "undefined"
+        ? global
+        : undefined;
+    }
+
+    const g = getGlobal();
+    g.logCRMEvent = logCRMEvent;
```

Comment:
@Rick-Kirkham Can you confirm if these statements and the getGlobal function should be replaced with an associate statement per PR https://github.com/OfficeDev/office-js-docs-pr/pull/3389? Thanks.

Reply:
> @Rick-Kirkham Can you confirm if these statements and the getGlobal function should be replaced with an associate statement per PR #3389? Thanks.

@Rick-Kirkham Following up on this question. Should we replace this with an associate statement? Thx.

Reply:
Yes.

> > @Rick-Kirkham Can you confirm if these statements and the getGlobal function should be replaced with an associate statement per PR #3389? Thanks.
> 
> @Rick-Kirkham Following up on this question. Should we replace this with an associate statement? Thx.

Yes.